### PR TITLE
zar-10-add-product-entity

### PR DESCRIPTION
**Pull request by Sidanton6006**

### CODE REVIEW

**Review by Mike-Opanasiuk — DISMISSED** [review]
Changes required.

File: Core/Entities/Product.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿using Core.Entities.Abstract;
+
+namespace Core.Entities;
+
+public class Product : BaseEntity
+{
+    public string Name;
+    public double Price;
```
Comment (Mike-Opanasiuk) [inline Core/Entities/Product.cs]: Change price type to decimal. Double type provides less precise way to save data.

**Review by byCodeXp — CHANGES_REQUESTED** [review]
- Product entity musy be named as ProductEntity
- Fields Name, Price, Description of ProductEntity must have get and set properties.
- You should add configuration for "Product" entity.

**Review by byCodeXp — APPROVED** [review]
Looks good

**Review by Mike-Opanasiuk — CHANGES_REQUESTED** [review]
Make ProductEnity **Name** field optional. Add migration after adding an entity.

**Review by Mike-Opanasiuk — APPROVED** [review]

**Pull request merged**